src/omotes_orchestrator/main.py

```diff
@@ -206,6 +213,7 @@ def stop(self) -> None:
         self.jobs_broker_if.stop()
         self.celery_if.stop()
         self.postgresql_if.stop()
+        self.postgres_job_manager.stop()
```

Comment:
`postgresql_if` is required by `postgres_job_manager`. Please flip them so `postgresql_if` is stopped after `postgres_job_manager` is done.